index.html

```diff
@@ -507,6 +517,16 @@ <h3 id="sec-vendor-prefix"><span class="secno">4.5 </span>Vendor Prefixes</h3>
     <li>and the above names are in ASCII.</li>
 </ul>
 
+<p>When vendor-neutral extensions to this specification are needed,
+either this specification can be updated accordingly, or an
+extension specification can be written that overrides the
+requirements in this specification. When someone applying this
+specification to their activities decides that they will recognize
+the requirements of such an extension specification, it becomes an
+<dfn id="other-applicable-specifications" title="other applicable specifications">applicable
+specification</dfn> for the purposes of conformance requirements in
+this specification.
+</p>
 

```

Comment:
Please remove this. This encourages :monkey: patching. Specially: 

> extension specification can be written that overrides the requirements in this specification.

Extensions to this specification should be done in *this* specification - as part of a errata or living standards model.